astro/src/content/blog/i-just-want-authentication-to-work.mdx

```diff
@@ -0,0 +1,51 @@
+
+![Napster app from Wikipedia](/img/blogs/i-just-want-authentication-to-work/napster.png)
+
+## The benefits of using an authentication service
+
+There are many benefits to using an authentication service. Authentication services can help you to:
+
+* Save time and money: Implementing and maintaining authentication can be time-consuming and expensive. Authentication services can save you the hassle of having to implement and maintain your own authentication system.
+* Improve security: Authentication services can help you to improve the security of your applications by using best practices and the latest security technologies.
+* Reduce the risk of compliance violations: Authentication services can help you to meet compliance requirements by providing features such as two-factor authentication and user provisioning.
+
+The last thing that I want is a call on Saturday night that someone hacked our password database that we built in house and didn't secure!
+
+![Meme you can't leak passwords if you don't store passwords, man pointing at head](/img/blogs/i-just-want-authentication-to-work/meme-leak.png)
+
+## Choosing an authentication service
+
+When choosing an authentication service, there are a few factors to consider:
+
+* Features: Consider the features that are important to you, such as two-factor authentication, user provisioning, and support for different authentication methods.
+* Pricing: Authentication services can vary in price. Compare the pricing of different services to find the best deal for your needs.
+* Ease of use: Consider how easy the authentication service is to use. You want a service that is easy to set up and manage.
+* Hosting: Do you need flexibility to host your own instance of an Authentication Provider or have them Host.
+
+## For developers who don't care about implementing authentication
+
+If you are a developer who doesn't care about implementing authentication, there are a few things that you can do to make your life easier:
+
+* Use an authentication service. There are many different authentication services available, both free and paid. Authentication services can save you the hassle of having to implement and maintain your own authentication system.
+* Use a framework or library that provides authentication functionality. There are many different frameworks and libraries available that provide authentication functionality. Using a framework or library can save you the time and effort of having to implement authentication yourself. If you are ready to get started checkout our [quickstarts](/docs/quickstarts/).
+* Consider when that Napster, wait I mean your app starts to grow and now you need authentication for mobile, desktop and web. You will want an Authentication provider that can handle each form of Authentication.
```

Comment:
```suggestion
* Consider that when Better Napster, wait I mean your app starts to grow, you will want an Authentication provider that can handle different forms of Authentication such as mobile, desktop and web.
```

Reply:
One of the things that I think is super useful about fusion auth is that it is super easy to implement other IdPs.  Google, Facebook, Apple, etc.

Another great thing about FusionAuth is that it can support multiple apps.  So not just Better Napster, but the Even Better Than That Napster.

I also think it is worth mentioning that FusionAuth can be deployed localy for development, hosted yourself or as a service and the code should work wherever you point it to.  Basically can do it all.